### ISSUE TITLE
scikit-learn version error

### ISSUE THREAD

**Issue by kojikojiprg** [Zhongdao/UniTrack#46]
I use `scikit-learn==1.1.2` on my project and I also use Unitrack as a submodule.
However, Unitrack uses `scikit-learn==0.22`, which version is too old.

So, import error is occured in `utils/mask.py` line.16.
```
from sklearn.metrics import jaccard_similarity_score
```
`jaccard_similarity_score` is deleted from `scikit-learn >= 0.23`.
(And it isn't used in `utils/mask.py`, so it can be deleted.)

Will you update scikit-learn?
If scikit-learn is updated, it might be some other errors will be occured.
